metaseq/models/transformer.py

```diff
@@ -561,10 +562,35 @@ def forward_embedding(
         incremental_state: Optional[Dict[str, Dict[str, Optional[Tensor]]]] = None,
     ):
         # embed tokens and positions
-        positions = None
+        if self.self_attn_doc_sep != -1:
```

Comment:
Can we move out this hard-coded sentinel value and assign it a variable name? Will be cleaner to refactor later on if that ever comes up.